apps/hash-integration-worker/src/linear-activities.ts

```diff
@@ -96,7 +98,25 @@ const createHashEntity = async (params: {
     },
   );
 
-  await HashEntity.createMultiple(
+  // TODO: allow creating policies alongside entity creation
+  //   see https://linear.app/hash/issue/H-4622/allow-creating-policies-alongside-entity-creation
+  const entityUuid = extractEntityUuidFromEntityId(entity.entityId);
+  await createPolicy(graphApiClient, params.authentication, {
+    name: `linear-synced-administer-entity-${entityUuid}`,
+    effect: "permit",
+    principal: {
+      type: "actor",
+      actorType: "machine",
+      id: params.authentication.actorId,
+    },
+    actions: ["viewEntity"],
+    resource: {
+      type: "entity",
+      id: entityUuid,
+    },
```

Comment:
This one I think the Graph needs to know about Linear types and the Linear bot's id, to be able to say it can do whatever with Linear types in webs.

Reply:
If the types are external, the types don't have a `WebId`. This I think will be the case for self-hosted instances?

Reply:
Yes, that should work. That should be the `createdById`. I guess in both cases that's `@linear`, right?